src/areaCalculator/Circle.java

```diff
@@ -0,0 +1,25 @@
+package areaCalculator;
+
+public class Circle implements Shape {
+
+    private double diameter;
```

Comment:
I think it is more natural to have radius instead of diameter in Circle class. From mathematical standpoint formula of area is `r * r * pi`, so now you need to divide it by 2